src/tools/perf/perftest_params.c

```diff
@@ -670,6 +684,23 @@ ucs_status_t parse_opts(struct perftest_context *ctx, int mpi_initialized,
             ctx->mpi = atoi(optarg) && mpi_initialized;
             break;
 #endif
+        case 'g': /* handles daemon-local long option as well */
+            status = ucs_sock_ipportstr_to_sockaddr(
+                optarg, DEFAULT_DAEMON_PORT, &ctx->dmn_info.dmn_local_addr);
+            if (status != UCS_OK) {
+                goto err;
+            }
+            break;
+        case 'G': /* handles daemon-remote long option as well */
+            status = ucs_sock_ipportstr_to_sockaddr(
+                optarg, DEFAULT_DAEMON_PORT, &ctx->dmn_info.dmn_remote_addr);
+            if (status != UCS_OK) {
+                goto err;
+            }
+            break;
+        case 'k': /* handles daemon-keep-running long option as well */
+            ctx->dmn_info.is_keep_running = 1;
```

Comment:
do we really need separate keep running option? I'd expect the daemon will keep running until the client finishes the batch file and send a FIN command that tells the daemon to exit

Reply:
Of course we can live without it. I added this option with the following ideas in mind:
1. (minor) It allows us to run multiple batches in a row, like a regular sender-receiver perftest
2. It allows us to run persistent daemon service, and execute multiple tests without restarting 2 daemon processes for each run. Personally I found this useful when manually testing performance on different message sizes. With this option I just have to restart sender and receiver, but not their daemons
3. It brings no complexity, but can be useful

But if we change design like you proposed in other comment, this option might be obsolete

Reply:
ok, I'll remove it